src/software/gui/generic_widgets/play_info/play_info_widget.cpp

```diff
@@ -16,8 +16,7 @@ void PlayInfoWidget::updatePlayInfo(const PlayInfo& play_info)
     }
 
     QString play_info_string =
-        QString("%1\n%2\n%3")
-            .arg(refbox_game_state_string, play_name_string, tactics_string);
+        QString("%1\n%2\n%3").arg(play_type_string, play_name_string, tactics_string);
```

Comment:
also here

Reply:
Whoops changed the text but not the variable. Fixed.